### PR TITLE
Gekimon day 1 ciel

### PR DESCRIPTION
**Pull request by Gekimon**

### CODE REVIEW

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 1]
```diff
@@ -205,15 +205,15 @@ But the teachers rely on you because you do that. I hope you don't let them get
...
 <Page64>
-Ciel-senpai rebukes Arihiko's lazy opinions.
+Ciel-senpai completely rebukes Arihiko's lazy opinions.//だらけきった有彦の意見を、正面からばっさりたしなめるシエル先輩。
 From the looks of it, he seems to be close to her in his own way.
 <Page65>
 "Senpai, do you do volunteer work every day by any chance?"
 "What, you don't know? Ciel-senpai here is such a handy person that she's called the true student council president."
 <Page66>
-"I know. She's the perfect third year who can solve anything, unlike the student council which is just a title at best."
+"I know. She's the perfect third year who can solve anything, unlike the so-called student council."//知ってる。肩書きだけの生徒会と違って、なんでも解決してくれる完璧な三年生だろ」If anyone has better ideas on how to phrase this
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:214]: Titular student council? Can't immediately come up with something better that doesn't involve pulling extra meaning out of nowhere, e.g. 
> Unlike that useless lot that are student council members in name only, she's a perfect third year who can solve anything.
> Unlike the members of the student council, who only [help students?] on paper, she's a perfect third year who can solve anything.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 2]
```diff
@@ -205,15 +205,15 @@ But the teachers rely on you because you do that. I hope you don't let them get
...
 <Page67>
-"Yeah. There's even a fan club made among the first years, and whenever something troublesome happens, even the teachers would say, 'Just talk to Ciel and you'll be fine'. Nowadays, Senpai could do just about anything and the teachers wouldn't complain."
+"Yeah. There's even a fan club made among the first years, and whenever something troublesome happens, even asking the teachers would get you nothing more than a 'Just talk to Ciel and you'll be fine'. Nowadays, Senpai could do just about anything and the teachers wouldn't complain."//「おう。一年じゃファンクラブまであるし、教師連中も厄介ごとが起こると『シエルに相談すれば安心だ』なんて<暖簾|のれん>に腕押し状態だからな。今じゃ先輩が何をしたって文句を言う教師もいないぐらいだ」
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:216]: Unrelated to your change, but don't love the 2x `even` here, maybe drop the first one.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 3]
```diff
@@ -316,16 +316,16 @@ To be honest, I don't feel comfortable talking about the Tohno family.
 <Page98>
 "But he even stays at your house, doesn't he? That's really close."
 <Page99>
-"That's not it, Senpai. He ran away just because he didn't wanna stay home during summer break. This bastard feels reserved towards the Arima family whom he's entrusted to."
+"That's not it, Senpai. He ran away just because he didn't wanna stay home during summer break. This bastard feels reserved towards the Arimas 'cause he's been left in their care."//「違いますよ先輩。遠野は長い休みになると家に居づらいってんで逃げてきただけです。このヤロウ、預けられたコトで有間の人たちに遠慮してんの」
 <Page100>
-"So, he comes to me when I have a room available. My sister likes him because he's good-looking, and so he comes to stay with no money at all even though he's filthy rich!"
+"So, he comes to me when I have a room available. My sister likes him because he's good-looking, and so he's bold enough to stay with us without even bringing anything, "//「で、部屋が空いてるオレのところに転がりこんでくるってワケ。こいつは外見がいいから姉貴にも気に入られちまってさ、厚かましくも手ぶらで泊まりにくるんだぜ！」手ぶら empty-handed, as in not bringing a gift to thank them, what he says two lines below
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:321]: sentence now ends in dangling comma

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 4]
```diff
@@ -338,12 +338,12 @@ Arihiko nods his head, seemingly convinced. I think this kind of overwhelming op
 I mean, did you not get along with your previous family...?"
 <Page106>
 "No, that's not true. This guy has no problems with the Arima family. They're all super nice people. Well, old man Arima is rather scary, though.
-And yet he doesn't accept their kindness and runs away to my place during days off. What's wrong with you, really?"
+And yet he rejects any talk of becoming their adopted child and runs away to my place during days off. What are you unhappy about, really?"//だって言うのにこいつは養子にならないかって話も断って、休みになればオレんところに逃げてきやがるんだ。ホントに何が不満なんだオマエは」
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:341]: wow 養子 got totally dropped here

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 5]
```diff
@@ -338,12 +338,12 @@ Arihiko nods his head, seemingly convinced. I think this kind of overwhelming op
...
 <Page107>
-"I'm not refusing their kindness. They've been good to me, I just don't want to burden them anymore."
+"There's no reason for me to be unhappy about anything. They've been good to me, I just don't want to burden them anymore."//「不満なんかある訳ないだろ。よくしてもらってるから、これ以上負担をかけたくないだけだ」
 <Page108>
 I finish my curry rice.
-All that's left now is a bowl of curry udon.
+All that's left now is a hefty bowl of curry udon.//残るはまるまる一人前のカレーうどんである。Hefty, plump, beefy, or just large?
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:346]: Hefty works for me

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt [hunk 6]
```diff
@@ -42,20 +42,20 @@ I didn't expect the Tohno family to call me back at all. Keiko-san, the rest of
 "You didn't think they would call you back?
 Even though you're the eldest son of the Tohno family?"
 <Page13>
-"Yes... to tell you the truth, my recovery was just a front. My father and I never got along. It's like he threw me out of the house because of my injury."
+"Yes... to tell you the truth, my recovery was just a front. My father and I never got along. It's like he threw me out of the house pretexting my injury to keep up appearances."//「はい。……本当の事を言ってしまうと、療養生活というのは建前なんです。俺と親父は犬猿の仲というヤツでして。怪我を理由に<体|てい>よく放り出されたようなものなんです」
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt:45]: Not sure `pretexting` is a word
> It's more like he threw me out of the house, using my injury as a pretext to keep up appearances.

**Review by rschlaikjer — CHANGES_REQUESTED** [review]

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 1 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:214]: I think so-called is fine

Comment (Gekimon) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:214]: Yeah that was my problem too.
>Unlike the members of the student council, who only help students on paper, she's a perfect third year who can solve anything.

That's a nice sentence, I like it, but it does add a little.
Well, if everyone is fine with "so-called", I'll keep it.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt (hunk 6 above)
Comment (Gekimon) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_2.txt:45]: Whoops haha, I got influenced by French it seems!

**Review by rschlaikjer — APPROVED** [review]

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_3.txt [hunk 7]
```diff
@@ -24,7 +24,7 @@ And you don't just wake up one day and suddenly master the art of running.
 "That's true. I don't understand either."
 <Page8>
 How can I put into words the psychology of being attracted to something beautiful?
-However, the situation where you cannot honestly say that something is beautiful is also wrong, isn't it?
+However, isn't there something wrong if we can't adequately describe beauty, or are unable to honestly admit that something is beautiful?//しかし、言葉にできない美しさというが、美しいものを素直に美しいと言えない状況も、また間違っているのではないだろうか。
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_3.txt:27]: > However, isn't there something wrong if we can't adequately describe beauty? If we can't honestly admit that something is beautiful?

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 8]
```diff
@@ -205,15 +205,15 @@ But the teachers rely on you because you do that. I hope you don't let them get
 <Page63>
 "It's okay, I do it because I like it.# Besides, the teachers are thinking about the school. I don't like the way you're talking about them, Inui-kun."
 <Page64>
-Ciel-senpai rebukes Arihiko's lazy opinions.
+Ciel-senpai completely rebukes Arihiko's lazy opinions.//だらけきった有彦の意見を、正面からばっさりたしなめるシエル先輩。
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:208]: Wouldn't ばっさり be more like "ruthlessly"?

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 9]
```diff
@@ -353,7 +353,7 @@ Senpai forces herself to act cheerful.
 This topic might have been fine for a long-time friend like Arihiko but for someone like Senpai, such a complex topic must've been bothersome at best.
 In fact, she seems to be fidgeting as if she is uncomfortable.
 <Page111>
-"Aah, I'm sorry, Senpai. But I've got to talk to Tohno about something in private so can you excuse us for a sec?"
+"Aah, I'm sorry, Senpai. But I've got to talk to Tohno about something in private, so could you please give us some space?"//「あー、ゴメン先輩。オレ、ちょっと遠野と内緒話があるからさ、席外してもらえない？」 "Can you excuse us" implies that they (Shiki & Arihiko) are the ones leaving right? If so, that's the contrary
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:356]: I think that the original leaves it ambiguous and sounds good, I think maybe revert.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt [hunk 10]
```diff
@@ -393,7 +393,7 @@ But this is one step short of horror! It's so cruel I can't even laugh!"
 <Page123>
 To tell the truth, even before I was involved in a car accident and hospitalized, I had a bad relationship with my father.
 The Tohno family had been a prominent family for a long time, and their way of life was far from the norm.
-All I can remember from those days was that huge, Western-style mansion. It was like that place had been frozen in time with period-accurate family principles to match.
+All I can remember from those days was that huge, Western-style mansion, as well as the pressure of our family rules which, just like the mansion, seemed frozen in time.//子供心に覚えているのは巨大な洋館と、その洋館に合わせて時代が止まっているかのような家訓の重圧だ。
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:396]: this line has gone through a lot of revisions and I think this is solid.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_1.txt [hunk 11]
```diff
@@ -38,7 +38,7 @@ This is the second time that I can't understand her thought process at all.
 "I'm sorry. That was kind of an arrogant thing to say, wasn't it?"
 "Yes. You were so cocky, I was unintentionally touched. It's kind of embarrassing to have a younger boy do that to you."
 <Page14>
-Despite her words, Senpai smiles happily.
+Despite her words, Senpai has an embarassed but happy grin on her face.//言葉とは裏腹に、先輩は嬉しそうに照れ笑いなどをする。
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_3_3_1.txt:41]: embarrassed, but happy

**Review by Avuxo — APPROVED** [review]
just some minor changes but mostly lgtm

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 8 above)
Comment (Gekimon) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:208]: I'm not too regarding on adverbs and adjectives, but yeah, "ruthlessly" is probably closer.

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt (hunk 9 above)
Comment (Gekimon) [reply script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt:356]: I'll revert then.

**Review by yk1512 — APPROVED** [review]

File: script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_1.txt [hunk 12]
```diff
@@ -25,6 +25,6 @@ It's a little early for school, and the city is still asleep. The station is qui
 Thank you for talking to me. If you see me at school, please don't hesitate to say hello."
 <Page8>
 Waving her hands with a smile, Senpai runs off.
```
Comment (arjan127) [inline script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_1.txt:27]: In my opinion, "waving" already implies she's using her hands to do so. How about "Waving with a smile on her face"? Sounds more natural to me.

**Pull request merged**